<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TalentTreeWidget</class>
 <widget class="QWidget" name="TalentTreeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>780</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1130</width>
    <height>780</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1130</width>
    <height>780</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="talenttreewidgetgroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1111</width>
     <height>681</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
            background: #001b35;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1051</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
            background: #002942;
			color: white;
        }</string>
    </property>
    <property name="title">
     <string>Данные персонажа</string>
    </property>
    <widget class="QLabel" name="labeConstPers">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
            color: white;
}</string>
     </property>
     <property name="text">
      <string>Персонаж</string>
     </property>
    </widget>
    <widget class="QComboBox" name="SelectCharactercomboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>431</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="labelConstClass">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>49</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color: white;
}</string>
     </property>
     <property name="text">
      <string>Класс</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPrinClass">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>20</y>
       <width>191</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color: white;
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelConstPoints">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>20</y>
       <width>49</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color: white;
}</string>
     </property>
     <property name="text">
      <string>Очки</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPrintPoints">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>20</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color: white;
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPrintLvL">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>20</y>
       <width>61</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color: white;
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelConstLvL_2">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>20</y>
       <width>49</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color: white;
}</string>
     </property>
     <property name="text">
      <string>Уровень</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxRang1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>1091</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
            background: #002942;
color: white;
        }</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButtonTalent1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>261</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTalent2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTalent3">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>20</y>
       <width>241</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTalent1Help">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTalent2Help">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>20</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTalent3Help">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>20</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxRang2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>1091</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
            background: #002942;
color: white;
        }</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButtonTalent4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>261</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTalent5">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTalent6">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>20</y>
       <width>241</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTalent4Help">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTalent5Help">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>20</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTalent6Help">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>20</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxSpecial1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>361</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
            background: #002942;
color: white;
        }</string>
    </property>
    <property name="title">
     <string>-</string>
    </property>
    <widget class="QPushButton" name="pushButtonSpecial1_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>261</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSpecial1Help_1">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSpecial1_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>261</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSpecial1_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>261</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSpecial1Help_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>130</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSpecialt1Help_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>230</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxSpecial2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>340</y>
      <width>351</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
            background: #002942;
color: white;
        }</string>
    </property>
    <property name="title">
     <string>-</string>
    </property>
    <widget class="QPushButton" name="pushButtonSpecialt2Help_1">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSpecial2Help_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>130</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSpecial2Help_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>230</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSpecial2_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSpecial2_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSpecial2_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxSpecial3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>340</y>
      <width>341</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
            background: #002942;
color: white;
        }</string>
    </property>
    <property name="title">
     <string>-</string>
    </property>
    <widget class="QPushButton" name="pushButtonSpecial3_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>241</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSpecial3_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>241</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSpecial3_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>241</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSpecial3Help_1">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSpecial3Help_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>130</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSpecial3Help_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>230</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: black;
	color: #00FF00;
}</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>700</y>
     <width>311</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: black;
	color: white;
	border: 4px solid white;
}
QPushButton:hover {
	color: #FF0E0E;
	border: 4px solid #FF0E0E;
}
QPushButton:pressed {
	background-color: #FF0E0E;
	border: 4px solid #FF0E0E;
}</string>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBack">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>700</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: black;
	color: white;
	border: 4px solid white;
}
QPushButton:hover {
	color: #FF0E0E;
	border: 4px solid #FF0E0E;
}
QPushButton:pressed {
	background-color: #FF0E0E;
	border: 4px solid #FF0E0E;
}</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonResetTalents">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>700</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: black;
	color: white;
	border: 4px solid white;
}
QPushButton:hover {
	color: #FF0E0E;
	border: 4px solid #FF0E0E;
}
QPushButton:pressed {
	background-color: #FF0E0E;
	border: 4px solid #FF0E0E;
}</string>
   </property>
   <property name="text">
    <string>Сбросить таланты</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1130</width>
     <height>780</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1130</width>
     <height>780</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>talenttreewidgetgroupBox</zorder>
  <zorder>pushButtonSave</zorder>
  <zorder>pushButtonBack</zorder>
  <zorder>pushButtonResetTalents</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
